### PR TITLE
[AeroFoil v2] A bug fixed as pointed out by Zolko-123

### PR DESCRIPTION
**Pull request by melwyncarlo**
Thank you for creating a pull request to contribute to FreeCAD-macros!
To integrate your macro please make sure the following steps are complete:

- [x] Please check this box if you're not submitting a new macro.   
- [ ] Are you submitting a new macro ?  
- [ ] Have you followed the ['How to submit a macro'](../README.md#how-to-submit-a-macro) section of the README.md ?  
- [ ] Your macro has a [Description](../README.md#macro-description) in its header.  
- [ ] Your macro has a [CamelCase name](../README.md#camelcase-macro-name).  
- [ ] Your macro is named [appropriately](../README.md#macro-name-specifics).  
- [ ] Your macro contains a [Metadata section](../README.md#macro-metadata) that immediately follows the header description.  
- [ ] Your macro is Python3/Qt5 compliant and tested on the latest FreeCAD stable and development releases.  
- [ ] You're including documentation on how your macro works (bonus: screenshots and/or video on the Wiki)  
- [ ] Commit message is [well-written](https://chris.beams.io/posts/git-commit/)  
- [ ] Commit message is titled in the following way `[MacroName] Short description`.
- [ ] Optional, write or update the changelog in the macro, from latest to oldest.

And please remember to update the Wiki with the features added or changed once this PR is merged.  
**Note**: If you don't have wiki access, then please mention your contribution on the [0.19 Changelog Forum Thread](https://forum.freecadweb.org/viewtopic.php?f=10&t=34586).

---

There was an issue with the newly added 'split' curve feature. It's been fixed now.

### CODE REVIEW

File: ObjectCreation/AeroFoil.FCMacro [hunk 1]
```diff
@@ -105,19 +105,19 @@
...
-__Comment__ = "AeroFoil creates airfoil curves and faces using pre-defined models, algebraic functions, and DAT or CSV Files"
-__Web__ = "https://github.com/melwyncarlo/AeroFoil"
-__Wiki__ = "http://www.freecadweb.org/wiki/index.php?title=Macro_AeroFoil"
-__Icon__ = "AeroFoil_UI_Files/AeroFoil.png"
-__Help__ = "Click on the AeroFoil button/macro, and follow the instructions in the subsequent dialog boxes."
-__Status__ = "stable"
-__Requires__ = "Freecad >= v0.17"
+__Title__         = "AeroFoil"
+__Author__        = "Melwyncarlo"
+__Version__       = "2.0.0"
```
Comment (galou) [inline ObjectCreation/AeroFoil.FCMacro:110]: Please upgrade to '2.1.0' or '2.0.1'.

Comment (melwyncarlo) [reply ObjectCreation/AeroFoil.FCMacro]: Done!

**Comment by galou** [comment]
I just want to inform @melwyncarlo that I move the macro icon beside the macro itself.

**Comment by melwyncarlo** [comment]
Okay, Mr. Galou. I'm working on a newer version. I'll added the changes then.

**Pull request merged**